docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md

```diff
@@ -346,7 +393,133 @@ txn_type      | debit
+INSERT INTO transactions VALUES (100, 10003, 'EU', 'checking', 400.00, 'credit');
+COMMIT;
+```
+
+If we attempt to run this while connected to a node in us-west-2:
+```sql
+BEGIN;
+```
+```
+BEGIN
+```
+```sql
+INSERT INTO transactions VALUES (100, 10002, 'EU', 'checking', 400.00, 'debit');
+```
+```
+ERROR:  Illegal state: Nonlocal tablet accessed in local transaction: tablet c5a611afd571455e80450bd553a24a64: . Errors from tablet servers: [Illegal state (yb/client/transaction.cc:284): Nonlocal tablet accessed in local transaction: tablet c5a611afd571455e80450bd553a24a64]
+```
+
+we get an error similar to the above. This is because, since we have created a transaction table for us-west-2, by default, YugabyteDB assumes that we want to run a transaction local to the region (using the transaction status table `system.transactions_us_west_2` which we created in Step 1), but such a transaction cannot modify data outside of us-west-2.
+
+However, if we instead connect to a node in eu-central-1 and run the exact same transaction:
+
+```sql
+BEGIN;
+```
+```
+BEGIN
+```
+```sql
+INSERT INTO transactions VALUES (100, 10002, 'EU', 'checking', 400.0, 'debit');
+```
+```
+INSERT 1 0
+```
+```sql
+INSERT INTO transactions VALUES (100, 10003, 'EU', 'checking', 400.0, 'credit');
+```
+```
+INSERT 1 0
+```
+```sql
+COMMIT;
+```
+```
+COMMIT
+```
+
+we are now able to finish and commit the transaction without error.
+
+Sometimes though, we might want to run a transaction that writes data to multiple regions, for example:
+
+```sql
+BEGIN;
+INSERT INTO transactions VALUES (100, 10004, 'US', 'checking', 400.00, 'debit');
+INSERT INTO transactions VALUES (200, 10005, 'EU', 'checking', 400.00, 'credit');
+COMMIT;
+```
+
+Running this transaction will fail whether we run it from us-west-2 or eu-central-1. The solution is to mark the transaction as a global transaction as follows:
+```sql
+SET force_global_transaction = TRUE;
+```
+```
+SET
+```
+```sql
+BEGIN;
+```
+```
+BEGIN
+```
+```sql
+INSERT INTO transactions VALUES (100, 10004, 'US', 'checking', 400.00, 'debit');
+```
+```
+INSERT 1 0
+```
+```sql
+INSERT INTO transactions VALUES (200, 10005, 'EU', 'checking', 400.00, 'credit');
+```
+```
+INSERT 1 0
+```
+```sql
+COMMIT;
+```
+```
+COMMIT
+```
+
+Setting `force_global_transaction = TRUE` tells YugabyteDB to use the `system.transactions` transaction table instead, which is presumed to be globally replicated, and lets us run distributed transactions that span multiple regions.
+
+{{< note title="Note" >}}
+While all distributed transactions can run without problems under `force_global_transaction = TRUE`, such transactions may incur significantly higher latencies when committing the transaction, since YugabyteDB must perform consensus across multiple regions to write to `system.transactions`. This should only be used when necessary, and `force_global_transaction = FALSE` (the default setting) should be used whenever possible.
```

Comment:
```suggestion
{{< note title="Global transaction latency" >}}
Only force global transactions when necessary. All distributed transactions _can_ run without problems under `force_global_transaction = TRUE`, but you may have significantly higher latency when committing the transaction, because YugabyteDB must achieve consensus across multiple regions to write to `system.transactions`. Whenever possible, use the default setting of `force_global_transaction = FALSE`.
```